src/matrix/kaldi-matrix.h

```diff
@@ -454,6 +454,12 @@ class MatrixBase {
   /// matrix and return normalizer (log sum of exponentials).
   Real ApplySoftMax();
 
+  /// Compute the Parametric rectified linear unit function; element by element,
+  /// *this = ParametricRelu(src).
+  /// The positive and negative values are scaled by updatable parameters.
+  /// (x > 0 ? alpha * x : beta * x)
```

Comment:
this is closer to the documentation I was talking about, but  a couple of issues:
  - the notion of 'updatable' is not relevant or meaningful at this level of code.
  - you need to be explicit about how the indexing works, e.g. document that src.NumCols() must equal a.Dim() and b.Dim(), that the dimension of *this must equal that of src, and that it implements:
  ' (*this)[i][j] = src[i][j] * (src[i][j] > 0 ? a[j] : b[j])'